overrides/config/ftbquests/quests/chapters/eapplied_energistics_2.snbt

```diff
@@ -1808,15 +1808,15 @@
 			description: [
 				"The &eCell Workbench&r allows you to &aAdd/Remove/Config&r &eUpgrade Cards&r from your &eStorage Cells&r and/or &ePartition&r them."
 				""
-				"Here's an example; Say you have an Cobblestone Generator and you want to pipe it into your &bStorage Network&r."
+				"Here's an example; Say you have a Cobblestone Generator and you want to pipe it into your &bStorage Network&r."
 				""
 				"But wait, you don't want to end up filling all of your &eStorage Cards&r with Cobblestone. What do you do?"
 				""
 				"&6Hint:&r Before you turn the page and check the answer, think of the &eUpgrade Cards&r we've introduced, could any of them achieve this task?"
 				"{@pagebreak}"
-				"You could install an &aOverflow Destruction Card&r onto an &eStorage Cell&r, but before you do that you must make sure that the &eStorage Cell&r will only accept Cobblestone otherwise you risk your &bStorage Network&r eating your items. But how do we do that?"
+				"You could install an &aOverflow Destruction Card&r onto a &eStorage Cell&r, but before you do that you must make sure that the &eStorage Cell&r will only accept Cobblestone otherwise you risk your &bStorage Network&r eating your items. But how do we do that?"

```

Comment:
```suggestion
				"You could install an &aOverflow Destruction Card&r into a &eStorage Cell&r, but before you do that you must make sure that the &eStorage Cell&r will only accept Cobblestone otherwise you risk your &bStorage Network&r eating your items. But how do we do that?"
```